core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -80,8 +80,7 @@ export class ExplorationHtmlFormatterService {
     var interactionChoices = null;
 
     if ('choices' in interactionCustomizationArgs) {
-      interactionChoices = interactionCustomizationArgs.choices.value.map(
-        choice => choice.getHtml());
+      interactionChoices = interactionCustomizationArgs.choices.value;
```

Comment:
Just to check ... this reads as something that was done for multiple choice input. But this PR isn't supposed to modify that, right?